cms-common/src/main/java/io/commercetools/sunrise/cms/CmsService.java

```diff
@@ -11,21 +11,12 @@
 @FunctionalInterface
 public interface CmsService {
 
-     /**
-     * Gets the content corresponding to the given CMS identifier for the first found given language.
-     * @param locales the list of locales used to translate the message
-     * @param cmsIdentifier identifier of the CMS entry field
-     * @return the {@code completionStage} of the content in the first found given language, or absent if it could not be found
-     */
-    CompletionStage<Optional<String>> get(final List<Locale> locales, final CmsIdentifier cmsIdentifier);
-
     /**
-     * Gets the content corresponding to the given CMS identifier for the first found given language.
-     * @param locales the list of locales used to translate the message
-     * @param cmsIdentifier identifier of the CMS content
-     * @return the {@code completionStage} of the content in the first found given language, or empty string if it could not be found
+     * Gets the page content corresponding to the given key.
+     * @param pageKey identifying the page
```

Comment:
@dariuszor the `pageKey` should be the `slug` we mentioned the other day. I just called it `pageKey` because I thought that name was less contentful-centric.

So as it was planned before, the developer should be able to initialize the ContentfulCmsService with the field name he uses to store the slug. The documentation for the Contentful module should state that this field should be of type slug, required and whatever other necessary restriction.

And the entry type is not important anymore for the interface. It's only page and field.